Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.9.1-alpine3.6 as unsee-builder
+FROM golang:1.9.2-alpine3.6 as unsee-builder
 COPY . /go/src/github.com/cloudflare/unsee
 
 ARG VERSION
```

Comment:
nit: it might be better to use a nodejs image here, rather than installing Node.js into a Go build image.

Reply:
but then I need to install go into nodejs image

Reply:
You can have multiple stages.

Reply:
That would be unrelated to version bumps here, feel free to open a ticket for that